code/citadel/cit_reagents.dm

```diff
@@ -153,14 +153,13 @@
 	if(prob(33))
 		if(M.getArousalLoss() >= 100 && ishuman(M) && M.has_dna())
 			var/mob/living/carbon/human/H = M
-			to_chat(H, "<span class='love'>Your libido is going haywire!</span>")
-			H.mob_climax(forced_climax=TRUE)
+			if(prob(50)) //Less spam
```

Comment:
I tested that and it almost never triggered.
It's prob(33) THEN prob(50), resulting in about a prob(16) total. If I made it prob(10), the end result would be a 3% chance to trigger :S.